test/unit/fixes.py

```diff
@@ -2,12 +2,76 @@
 # Licensed under the MIT License.
 
 import sklearn
+from sklearn.utils.validation import check_array
 
 
 def get_sklearn_expected_1d_message():
     # Handle change of message for sklearn
     if sklearn.__version__ < "0.23.0":
         expected_message = "bad input shape"
     else:
-        expected_message = "y should be a 1d array"
+        expected_message = "Data must be 1-dimensional"
     return expected_message
+
+
+def validate_data(estimator, X, **kwargs):
```

Comment:
I think these new ones are better off under a `fairlearn/utils/_fixes.py` file instead of here.